build.sbt

```diff
@@ -28,12 +28,13 @@ val Scala2 = "2.13.12"
 val Scala3 = "3.3.1"
 ThisBuild / scalaVersion        := Scala2
 ThisBuild / crossScalaVersions  := Seq(Scala2, Scala3)
+ThisBuild / tlJdkRelease        := Some(11)
```

Comment:
Is the intent to require JDK 11+ for Grackle?

Reply:
I think yes.